tests/test_cli_compile.py

```diff
@@ -592,7 +596,7 @@ def test_not_specified_input_file(runner):
     assert out.exit_code == 2
 
 
-def test_stdin(runner):
+def test_stdin(pip_conf, runner, from_line):
```

Comment:
```suggestion
def test_stdin(pip_conf, runner):
```

`from_line` could be deleted since it's not used in this test anymore